identity/backend/pom.xml

```diff
@@ -73,6 +73,22 @@
       <groupId>jakarta.annotation</groupId>
       <artifactId>jakarta.annotation-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-data-jpa</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.data</groupId>
+      <artifactId>spring-data-commons</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.data</groupId>
+      <artifactId>spring-data-jpa</artifactId>
```

Comment:
❓ Do we need this dependency here? As we include `spring-boot-starter-data-jpa` this dependency is already included https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-starters/spring-boot-starter-data-jpa/build.gradle#L11

Reply:
when I remove it, dependency analyze plugin is complaining:
```
[ERROR] Used undeclared dependencies found:
[ERROR]    org.springframework.data:spring-data-jpa:jar:3.2.5:compile
[INFO] Add the following to your pom to correct the missing dependencies: 
[INFO] 
<dependency>
  <groupId>org.springframework.data</groupId>
  <artifactId>spring-data-jpa</artifactId>
  <version>3.2.5</version>
</dependency>
```

Reply:
Interesting, and I guess removing the starter would also produce a similar issue?

Reply:
to be precise another kind of issue :D